### ISSUE TITLE
Double license of nvd3

### ISSUE THREAD

**Issue by nterray** [novus/nvd3#1838]
Hello,

Nvd3 is licensed under Apache License, v 2.0. The license of our project is GPLv2 (which is incompatible with Apache 2.0) and therefore we cannot use nvd3, unfortunately. We can use d3.js which is BSD though.

Do you think it would be possible to dual license nvd3, so that users can choose between Apache 2.0 or a more permissive license like MIT or BSD? It would be a huge benefit for us.

Thanks

**Comment by liquidpele** [comment]
No.   However, I don't see any reason why you can't use the library unless you're trying to essentially re-package and sell it...

**Comment by nterray** [comment]
Thanks for your answer.

Yes we must package it, so we're doomed. :disappointed: 

Just for curiosity, may I ask you why you think that the double licensing is not a viable option for you?
